src/core/surface/call.c

```diff
@@ -247,6 +247,9 @@ static int fill_send_ops(grpc_call *call, grpc_transport_op *op);
 static void execute_op(grpc_call *call, grpc_transport_op *op);
 static void recv_metadata(grpc_call *call, grpc_metadata_batch *metadata);
 static void finish_read_ops(grpc_call *call);
+static grpc_call_error grpc_call_cancel_with_status_internal(
```

Comment:
remove the grpc_ prefix for static functions... suggest 'cancel_with_status(....)'